libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -1091,7 +1091,9 @@ void PbftManager::secondFinish_() {
     LOG(log_dg_) << "Node " << node_addr_ << " broadcast next votes for previous round. In period " << period
                  << ", round " << round << " step " << step_;
     if (auto net = network_.lock()) {
-      net->getSpecificHandler<network::tarcap::VotesSyncPacketHandler>()->broadcastPreviousRoundNextVotesBundle();
```

Comment:
but it actually does broadcast only previous round next votes or not ? broadcastNextVotesBundle evokes it sends also current round next votes

Reply:
After this change it broadcasts bundle that we are passing to it, so I think new naming `broadcastNextVotesBundle` is better

Reply:
yes so what was this change actually good for when we dont pass there anything else than previous round next votes ? + it is always called with `next_votes_manager_->getNextVotes()`, why not to call it inside the function ?

Reply:
There was no `NextVotesManager` in this packet handler before the latest rebase. And it wasn't worth to add it there just because of this one call. So I renamed it back and moved this call back inside of this method